docs/vscode.md

```diff
@@ -2,4 +2,33 @@
 title: Running from VS Code
 ---
 
-TODO: add docs
+* toc
+{:toc}
+
+## Install the VS Code extensions
+
+To use the Dart DevTools from VS Code you will need the [Dart extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.dart-code). If you're debugging Flutter applications you should also install the [Flutter extension](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter).
+
+## Start an application to debug
+
+Start a debug session for your application by opening the root folder of your project (the one containing `pubspec.yaml`) in VS Code and clicking **Debug > Start Debugging** (`F5`).
+
+## Launch the DevTools
+
+Once the debug session is active and the application has started, the **Dart: Open DevTools** command will become available in the VS Code command palette:
+
+<img src="{{ "/assets/images/vscode/command.png" | relative_url }}" width="700" height="115" />
```

Comment:
Is this pattern necessary for all url references?

```
{{ "/assets/images/vscode/command.png" | relative_url }}
```

It is a bit verbose, and the nice thing about markdown docs are that they're generally simple to author.

Reply:
Define necessary :)

If the user can get to the page both with and without a trailing slash, I think we need absolute or root-relative paths. It doesn't seem possible on live in my testing, but does with `jekyll serve`, though I'm not sure why - on my blog everything ends up with trailing slashes, and it seems like [this has changed](http://marcgg.com/blog/2016/03/14/github-pages-jekyll/) unexpectedly in the past.

We could removing `| relative_url` and manually put a root-relative path, but it means hard-coding the `/devtools/` prefix and I don't know if we'd want that (we could also use `{{ site.base }}` but that just seems very similar to what we have).

LMK what you think would be best. If we think the trailing slashes will never work, I can just put relative paths in directly - I often default to over-cautious =)

Reply:
> but it means hard-coding the `/devtools/` prefix

That sounds reasonable; you have more jekyll experience than me however, so I'll defer to whichever you think best.